service/src/test/resources/ehr/mapper/observation/expected-output-observation-statement-25.xml

```diff
@@ -0,0 +1,20 @@
+<component typeCode="COMP" >
+    <ObservationStatement classCode="OBS" moodCode="EVN">
+        <id root="394559384658936" />
+        <statusCode code="COMPLETE" />
+        <effectiveTime>
+            <center value="20100714163251"/>
+        </effectiveTime>
+        <availabilityTime value="20100113152950"/>
+        
+        <pertinentInformation typeCode="PERT">
+            <sequenceNumber value="+1" />
+            <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                <text>Component(s): [Code: Allergy to penicillin Code Value: test-code Test display  Interpretation Code: Example without userSelected Interpretation Text: Example interpretation text 1 Range: 20 test-unit 30 test-unit] [Code: BMD - Left Femur String Value: test-string Interpretation Code: Example with userSelected Interpretation Text: Example interpretation text 2] [String Value: Example of a valueString] [Quantity Value: 7.2 g/dl] Example comment</text>
```

Comment:
I am not sure the Code Value should be in observation pertinentInformation. Also there is double space after.

Reply:
For Observation.component we handle the value[x] field as same for Observation (uncategorised).
And for this translation, have translated to valueCodeableConcept to Code Value string.
The double space is due to how PertinentInformationObservationMapper.mapObservationValueToPertinentInformation builds strings - using spaces. 
I did not write this method, so think the easiest solution is to call trim() after to avoid the double space if you agree?